src/JK/WebsiteBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,12 @@
+parameters:
+    jk.website.controller_index.class: JK\WebsiteBundle\Controller\IndexController
+
+services:
+    jk.common.controller_abstract:
```

Comment:
You already defined this service in `CommonBundle`.
Also, in bundle's service configuration file you have to define services, which belongs only to this bundle. Here I see definition of service from another bundle